lightning/src/ln/channel.rs

```diff
@@ -3613,26 +3833,73 @@ pub(crate) fn per_outbound_htlc_counterparty_commit_tx_fee_msat(feerate_per_kw:
 	}
 }
 
+pub(super) fn calculate_our_funding_satoshis(
+	is_initiator: bool, funding_inputs: &[(TxIn, TransactionU16LenLimited)],
+	funding_feerate_sat_per_1000_weight: u32, holder_dust_limit_satoshis: u64,
+) -> Result<u64, APIError> {
+    use bitcoin::WScriptHash;
```

Comment:
Any particular reason this `use` is local here?

Reply:
Nope, just rust-analyzer being weird :)